src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java

```diff
@@ -1697,4 +1736,157 @@ void reportTest() throws IOException {
 
         assertEquals(refLogExport, logExport);
     }
+
+    @Test
+    void testSecurityAnalysisWithOperatorStrategy() {
+        MatrixFactory matrixFactory = new DenseMatrixFactory();
+        GraphConnectivityFactory<LfBus, LfBranch> connectivityFactory = new NaiveGraphConnectivityFactory<>(LfBus::getNum);
+        securityAnalysisProvider = new OpenSecurityAnalysisProvider(matrixFactory, connectivityFactory);
```

Comment:
to fix by reusing the runSecurityAnalysis with a different connectivity factory

Reply:
I don't understand your comment. If I use another factory, it is not working...